tests/test-coreos.sh

```diff
@@ -34,8 +34,8 @@ fetch_repo() {
 
 fetch_latest_rhcos_image() {
   local image_path=$1
-  local image_url="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/latest/"
-  local latest_image_gz=$(curl ${image_url} | grep -o rhcos-[0-9].[0-9].[0-9]\\+-x86_64-qemu.x86_64.qcow2.gz | head -1)
+  local image_url="https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest"
+  local latest_image_gz=$(curl ${image_url}/ | grep -Po "rhcos-([[:digit:]]).([[:digit:]]){1,2}.([[:digit:]]{1,2})-x86_64-qemu.x86_64.qcow2.gz" | head -1)
```

Comment:
we can replace this with `kcli download image rhcos45` or whatever version we want

Reply:
After discussing offline we decided to stick to current way